.github/workflows/cloud-workflow.yml

```diff
@@ -0,0 +1,175 @@
+---
+name: cloud-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  insync-checkin:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Run build.py
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --generate
+            sudo chown -R $USER $MAGMA_ROOT/*
+            git add .
+            git status
```

Comment:
While we're here, can I make a small request for improvement to this job: can we pipe `git status`'s output to the Slack notification and make it viewable from the job

Rationale: in the past, when our generated code gets into a weird state, it's been hard to debug this failure without knowing which files were changed

Reply:
I am inclined to tackle this separately. We will need to think about the way to concatenate this with the current commit message in a readable format.

Reply:
tracking it here https://github.com/magma/magma/issues/7992